airgun/views/hostcollection.py

```diff
@@ -185,7 +183,9 @@ def get_action_button(self, name):
         """Return an action button by it's name"""
         action_button = getattr(self, name)
         if not isinstance(action_button, ActionsDropdown):
-            raise ValueError(f'Action with name: "{name}" does not exists')
+            raise TypeError(
```

Comment:
I think `ValueError` is more correct it seems, as TypeError would come if we are try to type something

Reply:
TypeError is more accurate since we're checking the type of the action_button as the error condition. See: https://beta.ruff.rs/docs/rules/type-check-without-type-error

Reply:
is that sense make sense